src/ramses_tx/parsers.py

```diff
@@ -1531,6 +1531,8 @@ def parser_22e9(payload: str, msg: Message) -> Mapping[str, float | str | None]:
 
 # fan_speed (switch_mode), HVAC
 def parser_22f1(payload: str, msg: Message) -> dict[str, Any]:
+    # ClimaRad VenturaV1x HRU does not send 22F1 for speed, uses 22F4 for mode + step
+
```

Comment:
Going forward - put these comments in the test log files

Otherwise, teh codebase is going to bloat